### ISSUE TITLE
[TESTING] Ultralight Stability

### ISSUE THREAD

**Issue by 1zun4secondary** [CCBlueX/LiquidBounce#1026]
### Describe your feature request.

Our biggest issue with Ultralight is the lack of GPU implementation and the stability. Native implemenation can cause the process to suddently exit without any debug information. This can lead to hard times debugging issues. Nextgen have already introduced a thread-safe way to interact with Ultralight but it seems this is not enough to provide a reliable way of using Ultralight without it crashing at the random moments.

**Comment by 1zun4** [comment]
I have now recorded multiple hours of footages of playing with Nextgen and how should I say... I have not experienced any crashes so far. I am not sure if we accidently fixed it or if I am lucky at the moment. Do you know? @mems01 
Of course we are still missing GPU rendering but I don't think this is 1.0.0 blocking, because anyone with a decent CPU can run the UI very well.

**Comment by 1zun4** [comment]
https://www.ibm.com/docs/en/sdk-java-technology/8?topic=jni-debugging

**Comment by ghost** [comment]
Yes, it still does crash for me.

**Comment by 1zun4** [comment]
Current status: Seems to be stable enough.

For the crash issue (on Linux machine)
```
Problematic frame:
# C  [libUltralightCore.so+0x2cee6]  ultralight::BitmapSurface::bitmap()+0x16
``` 
After applying the following fix https://github.com/CCBlueX/LiquidBounce/commit/a7511b0d2062c4177c22938a214accde152a6b32 there have been no incidents of the client crashing due to this bug for 2 weeks.

On my Linux installation, there have been no crashes since applying this fix.
Also on Windows, stability has improved with people running the client for 8+ hours straight, so it seems to be fixed there as well.

Another very rare issue, which has only happened once so far, is the following:
```
[08:00:13] [Render thread/ERROR] (LiquidBounce) Exception while executing handler.
 java.lang.IllegalArgumentException: newPosition > limit: (23496 > 16)
	at java.nio.Buffer.createPositionException(Buffer.java:341) ~[?:?]
	at java.nio.Buffer.position(Buffer.java:316) ~[?:?]
	at java.nio.ByteBuffer.position(ByteBuffer.java:1516) ~[?:?]
	at java.nio.MappedByteBuffer.position(MappedByteBuffer.java:321) ~[?:?]
	at java.nio.MappedByteBuffer.position(MappedByteBuffer.java:73) ~[?:?]
	at net.ccbluex.liquidbounce.base.ultralight.impl.renderer.CpuViewRenderer.render(CpuViewRenderer.kt:105) ~[main/:?]
	at net.ccbluex.liquidbounce.base.ultralight.ViewOverlay.render(ViewOverlay.kt:127) ~[main/:?]
	at net.ccbluex.liquidbounce.base.ultralight.UltralightEngine.render(UltralightEngine.kt:169) ~[main/:?]
	at net.ccbluex.liquidbounce.base.ultralight.hooks.UltralightIntegrationHook$overlayRenderHandler$1.invoke(UltralightIntegrationHook.kt:40) ~[main/:?]
	at net.ccbluex.liquidbounce.base.ultralight.hooks.UltralightIntegrationHook$overlayRenderHandler$1.invoke(UltralightIntegrationHook.kt:39) ~[main/:?]
	at net.ccbluex.liquidbounce.event.EventManager.callEvent(EventManager.kt:88) ~[main/:?]
	at net.minecraft.client.gui.hud.InGameHud.handler$zpf000$liquidbounce$hookRenderEvent(InGameHud.java:2441) ~[minecraft-merged-f333d3c2e5-1.20.1-net.fabricmc.yarn.1_20_1.1.20.1+build.10-v2.jar:?]
	at net.minecraft.client.gui.hud.InGameHud.render(InGameHud.java:258) ~[minecraft-merged-f333d3c2e5-1.20.1-net.fabricmc.yarn.1_20_1.1.20.1+build.10-v2.jar:?]
``` 

THIS IS NOT A ULTRALIGHT BUG, but just uncareful usage of the NioBuffer that we should easily be able to fix.
https://github.com/CCBlueX/LiquidBounce/blob/5a5b86f25b616242cd315e0db97945416b5aa06c/src/main/kotlin/net/ccbluex/liquidbounce/base/ultralight/impl/renderer/CpuViewRenderer.kt#L105
